src/ogma/index.ts

```diff
@@ -39,19 +39,26 @@ interface AddItemOptions {
 }
 
 export class LKOgma extends Ogma<LkNodeData, LkEdgeData> {
-  private _reactive: RxViz;
+  private _reactive?: RxViz;
   public LKStyles!: StylesViz;
   public LKCaptions!: CaptionsViz;
-  public LKTransformation: TransformationsViz;
+  public LKTransformation!: TransformationsViz;
   // Trigger an event with node category changes
-  public nodeCategoriesWatcher: NonObjectPropertyWatcher<LkNodeData, LkEdgeData>;
+  public nodeCategoriesWatcher!: NonObjectPropertyWatcher<LkNodeData, LkEdgeData>;
   // Trigger an event with edge type changes
-  public edgeTypeWatcher: NonObjectPropertyWatcher<LkNodeData, LkEdgeData>;
-  public store: OgmaStore;
+  public edgeTypeWatcher!: NonObjectPropertyWatcher<LkNodeData, LkEdgeData>;
+  public store!: OgmaStore;
+  private _configuration: IOgmaConfig;
 
   constructor(configuration: IOgmaConfig) {
```

Comment:
You can set the configuration directly in the constructor: 
```suggestion
  constructor(private _configuration: IOgmaConfig) {
```